install/tools/ipa-pki-retrieve-key.in

```diff
@@ -2,41 +2,53 @@
 
 from __future__ import print_function
 
+import argparse
 import os
-import sys
-import traceback
 
+from ipalib import api
 from ipalib import constants
-from ipalib.config import Env
+from ipalib.krb_utils import krb5_format_service_principal_name
 from ipaplatform.paths import paths
 from ipaserver.secrets.client import CustodiaClient
 
 
 def main():
-    env = Env()
-    env._finalize()
+    api.bootstrap()
 
-    keyname = "ca_wrapped/" + sys.argv[1]
-    servername = sys.argv[2]
+    parser = argparse.ArgumentParser("ipa-pki-retrieve-key")
+    parser.add_argument("keyname", type=str)
+    parser.add_argument("servername", type=str)
+
+    args = parser.parse_args()
+    args.keyname = "ca_wrapped/{}".format(args.keyname)
```

Comment:
Nit: not a fan of clobbering the original value here.  It would be better to just have `keyname = ...` and refer to `keyname` later.

Reply:
Fixed